### PR TITLE
S01 us11 onglets cours

### PR DESCRIPTION
**Pull request by ChristianOlmedo**

### CODE REVIEW

File: src/View/User/Includes/_course_tab.html.twig [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{% block course_tab %}
+<div class="course-tab-section col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+    <div class="tab_title_text">
+        <p class="tab_title">
+            CONDITIONS
```
Comment (bouboumael) [inline src/View/User/Includes/_course_tab.html.twig:5]: your forgot to use our fonts

File: src/View/User/Includes/_course_tab.html.twig [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        </p>
+        <p class="tab_text">
+            </br> Lorem ipsum dolor sit amet. Sed doloribus id eligendi rerum eos impedit dolore eos excepturi quisquam et corporis Quis At sunt voluptatem ex ullam dolorum. Et corrupti tempore id voluptatibus atque eos itaque enim non impedit laudantium est excepturi sunt et quia saepe! Non accusantium quia id culpa iure sed fugiat optio a reiciendis ipsum eum sunt nostrum.
```
Comment (bouboumael) [inline src/View/User/Includes/_course_tab.html.twig]: same has before and repeat this for the of your user story

**Review by bouboumael — CHANGES_REQUESTED** [review]

File: src/View/User/Includes/_course_tab.html.twig (hunk 1 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig:5]: Done

File: src/View/User/Includes/_course_tab.html.twig (hunk 2 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig]: Done

**Review by bouboumael — DISMISSED** [review]

File: src/View/User/Includes/_course_tab.html.twig (hunk 2 above)
Comment (LucyMerlier) [inline src/View/User/Includes/_course_tab.html.twig]: is br useful here?

File: src/View/User/Includes/_course_tab.html.twig [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                <div class="img_course">
+                    <img src="" alt="Cheval Privilège photo">
+                </div>
+                Horaires: </br>
+                Tarif: </br>
+            </div>
+        </div>
+    </div>
+</div>
+{% endblock %}
```
Comment (LucyMerlier) [inline src/View/User/Includes/_course_tab.html.twig]: class naming : good practice is to use dashes ("-"), not underscores.
also, avoid leaving empty src attributes for images : try using lorem picsum, for example

File: src/View/User/Includes/_course_tab.html.twig [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            Cheval
+        </button>
+        <div class="collapse" id="collapse4">
+            <div class="card card-body">
+                Niveau minimum: Galop 3 </br>
+                <div class="img_course">
+                    <img src="" alt="Cheval Privilège photo">
+                </div>
+                Horaires: </br>
+                Tarif: </br>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_course_tab.html.twig]: why br here?

File: src/View/User/Includes/_course_tab.html.twig [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            </div>
+        </div>
+        <button class="btn_course" type="button" data-bs-toggle="collapse" data-bs-target="#collapse4">
+            Cheval
+        </button>
+        <div class="collapse" id="collapse4">
+            <div class="card card-body">
+                Niveau minimum: Galop 3 </br>
+                <div class="img_course">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_course_tab.html.twig]: there's a bootstrap class for card images ;)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/User/Includes/_course_tab.html.twig (hunk 2 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig]: Indeed, it's useless, i removed them

File: src/View/User/Includes/_course_tab.html.twig (hunk 4 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig]: Probably br is not a good practice ?? I removed them to place my text in p tags, the goal is to seperate horaires and tarifs

File: src/View/User/Includes/_course_tab.html.twig (hunk 5 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig]: Done

File: src/View/User/Includes/_course_tab.html.twig (hunk 3 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig]: Done

**Review by NEL45 — DISMISSED** [review]
All good

File: public/assets/css/course_tab.css [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    width: var(--width);
+    color: var(--dark);
+    border: none;
+}
+
+.btn-course {
+    --margin: 0.5em;
+    --height: 3em;
+    --width: 13em;
+    --border-radius: 2em;
```
Comment (LucyMerlier) [inline public/assets/css/course_tab.css:54]: I still feel like you have a lot of different values for padding, margins, etc... maybe you should change that in the future and use multiples of one single value (4 and 8px are commonly used for paddings and margins)

File: src/View/User/Includes/_course_tab.html.twig [hunk 7]
```diff
@@ -0,0 +1,90 @@
+{% block course_tab %}
+<div class="course-tab-section col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+    <div class="tab-title-text">
+        <p class="tab-title">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_course_tab.html.twig]: if this is a title, why use the p tag? an h* seems more appropriate

File: src/View/User/Includes/_course_tab.html.twig [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+        </button>    
+        <div class="collapse" id="collapse1">
+            <div class="card" style="width: 18rem;">
+                <img src="https://picsum.photos/id/1024/300/300" class="card-img-top" alt="Baby Poney Photo">
+                <div class="card-body">
+                    <p class="card-text">
+                    <p> Âge: à partir de 18 mois </p>
+                    <p> Horaires: </p>
+                    <p> Tarif: </p>
+                    </p>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_course_tab.html.twig]: this looks like a list ;)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/User/Includes/_course_tab.html.twig (hunk 7 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig]: Done

File: src/View/User/Includes/_course_tab.html.twig (hunk 8 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_course_tab.html.twig]: I don't know why is that a problem, but i changed to <li> tags

**Review by NEL45 — DISMISSED** [review]

File: src/View/User/Includes/_course_tab.html.twig [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+        </button>    
+        <div class="collapse" id="collapse1">
+            <div class="card" style="width: 18rem;">
+                <img src="https://picsum.photos/id/1024/300/300" class="card-img-top" alt="Baby Poney Photo">
+                <div class="card-body">
+                    <p class="card-text">
+                    <li> Âge: à partir de 18 mois </li>
+                    <li> Horaires: </li>
+                    <li> Tarif: </li>
+                    </p>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_course_tab.html.twig]: invalid html : li tags nested in p tags

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
invalid html

**Review by NEL45 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]
seems ok

**Pull request merged**